app/controllers/users_controller.rb

```diff
@@ -1,3 +1,29 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
dùng find_by thay vì find nè, thêm code xử lý trường hợp user bị nil như bên dưới nhé 
`return @user`
`flash[:warning] = "User not found"`
`redirect_to root_path`